js/ui/appDisplay.js

```diff
@@ -373,7 +373,7 @@ const AllView = new Lang.Class({
 
     _init: function() {
         this.parent({ usePagination: true }, null);
-        this._scrollView = new St.ScrollView({ style_class: 'all-apps',
+        this._scrollView = new St.ScrollView({ style_class: 'all-apps vfade',
```

Comment:
It's an upstream issue, just didn't have time to report it yet. From what I can see we also added ' vfade' in eos-desktop in the past, so it looks like it's been there for a while.